definitions/checks/disk/available_space_var.rb

```diff
@@ -0,0 +1,26 @@
+module Checks
+  module Disk
+    class AvailableSpaceCandlepin < ForemanMaintain::Check
+      metadata do
+        label :available_space_cp
+        description 'Check to make sure /var/lib/candlepin has enough space'
+        tags :pre_upgrade
+        confine do
+          feature(:candlepin) && check_min_version('candlepin', '3.1')
+        end
+      end
+
+      MAX_USAGE_IN_PERCENT = 90
+
+      def run
+        assert(enough_space?, "System has more than #{MAX_USAGE_IN_PERCENT}% space used"\
```

Comment:
Is it good idea to mention bugzilla no 1898605 in message as its public ? I think it will help user to know why this check is implemented and maybe track the bugzilla for progress ?

Reply:
Nice idea, should I display the URL or is it enough to mention `Bugzilla #1898605`?

Reply:
Do I need to add a newline into the message, because it is rather long right now?

Reply:
I vote for full bugzilla URL on new line !